models/issue_comment.go

```diff
@@ -116,6 +118,8 @@ type Comment struct {
 	Assignee         *User `xorm:"-"`
 	OldTitle         string
 	NewTitle         string
+	OldBranch        string
```

Comment:
The length default is `VARCHAR(255)`. Is this enough for git branch name?

Reply:
`string` is used in the PullRequest model (see _HeadBranch_ and _BaseBranch_). I believe that should be fine?